dataclients/kubernetes/kube.go

```diff
@@ -888,6 +888,17 @@ func (c *Client) ingressToRoutes(items []*ingressItem) ([]*eskip.Route, error) {
 			}
 		}
 
+		// parse pathmode from annotation or fallback to global default
```

Comment:
i think we should not fallback to the global default, but reject these rules instead

Reply:
I decided with @mikkeloscar that it makes more sense to have the default

1. All the annotation handling is right now like this, so it should follow the same style.
2. it's more safe for users and we do not want to block route udaptes, because someone specified a wrong ingress

If we would stop the route updates, which the return error would do, then all new pods will not get traffic (skipper loadbalancer use endpoints and has to update the routes).